### ISSUE TITLE
[master] XRC20 token transfer on explorer

### ISSUE THREAD

**Issue by zjshen14** [iotexproject/iotex-explorer#731]
It may depend on https://github.com/iotexproject/iotex-analytics/issues/48

**Comment by puncsky** [comment]
contract(address:String!,numPerPage:Int!,page:Int!):[Contract]

type Contract{
    hash:String!
    timestamp:String!
    from:String!
    to:String!
    quantity:String!
}

**Comment by puncsky** [comment]
add a page for `https://iotexscan.io/token/io1hp6y4eqr90j7tmul4w2wa8pm7wx462hq0mg4tw`

**Comment by lizhefeng** [comment]
We have some renaming for thisAPI. Will give you the update here.

**Comment by puncsky** [comment]
any update on this?

**Comment by nmhung1210** [comment]
Is it possible to query xrc20 transfer in common (like https://etherscan.io/tokentxns)? Current, I see this API query specifically for a contract address only.

**Comment by lizhefeng** [comment]
```
query{
  xrc20(address:"io14j96vg9pkx28htpgt2jx0tf3v9etpg4j9h384m",
numPerPage:10,page:19){
  hash,
  from,
  to,
  quantity,
  timestamp,
  }
}
```
Just changed contract to xrc20.

**Comment by lizhefeng** [comment]
@nmhung1210 Might be possible. Let me double check.

**Comment by nmhung1210** [comment]
@lizhefeng we would need an API to return token transfer listed for a wallet address too (like https://etherscan.io/address/0xbc55ddde7e65cca2a662c76fed091c682ed105f2#tokentxns). Could you please help check if it's feasible as well?

**Comment by lizhefeng** [comment]
@nmhung1210 Please create two separate tickets for queries of total list of XRC20 transfers and list of XRC20 transfers by address.

**Comment by nmhung1210** [comment]
Closed in favor of #1028
